cli/main.go

```diff
@@ -109,12 +110,14 @@ Commands:
 		args := flag.Args()[1:]
 		seq := false
 		seqDigits := 6
+		datetime := false
 
 		createFlagSet := flag.NewFlagSet("create", flag.ExitOnError)
 		extPtr := createFlagSet.String("ext", "", "File extension")
 		dirPtr := createFlagSet.String("dir", "", "Directory to place file in (default: current working directory)")
 		createFlagSet.BoolVar(&seq, "seq", seq, "Use sequential numbers instead of timestamps (default: false)")
 		createFlagSet.IntVar(&seqDigits, "digits", seqDigits, "The number of digits to use in sequences (default: 6)")
+		createFlagSet.BoolVar(&datetime, "datetime", datetime, "Generate migrations where the version is yyyymmddhhmmss")
```

Comment:
The Go time format string specifier wasn't obvious to me when I first encountered it so it's worth documenting it. e.g. the format string provides an example of what the reference time looks like instead of specifying time element formatters.